app/code/Magento/ReviewGraphQl/Model/DataProvider/CustomerReviewsDataProvider.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ReviewGraphQl\Model\DataProvider;
+
+use Magento\Review\Model\ResourceModel\Review\Product\Collection;
+use Magento\Review\Model\ResourceModel\Review\Product\CollectionFactory;
+use Magento\Review\Model\Review\AddRatingVotesToCustomerReviews;
+
+/**
+ * Provides customer reviews
+ */
+class CustomerReviewsDataProvider
+{
+    /**
+     * @var CollectionFactory
+     */
+    private $collectionFactory;
+
+    /**
+     * @var AddRatingVotesToCustomerReviews
+     */
+    private $addRatingVotesToCustomerReviews;
+
+    /**
+     * CustomerReviewsDataProvider constructor.
+     *
+     * @param CollectionFactory $collectionFactory
+     * @param AddRatingVotesToCustomerReviews $addRatingVotesToCustomerReviews
+     */
+    public function __construct(
+        CollectionFactory $collectionFactory,
+        AddRatingVotesToCustomerReviews $addRatingVotesToCustomerReviews
+    ) {
+        $this->collectionFactory = $collectionFactory;
+        $this->addRatingVotesToCustomerReviews = $addRatingVotesToCustomerReviews;
+    }
+
+    /**
+     * Get customer reviews
+     *
+     * @param int $customerId
+     * @param int $currentPage
+     * @param int $pageSize
+     *
+     * @return Collection
+     */
+    public function getData(int $customerId, int $currentPage, int $pageSize): Collection
```

Comment:
Please, use alias for the `Collection` here. So we have `ProductReviewsCollection` or so